spinoso-string/src/enc/utf8/owned.rs

```diff
@@ -146,6 +147,16 @@ impl Utf8String {
         self.try_push_codepoint(int)
     }
 
+    #[inline]
+    pub fn codepoints(&self) -> Result<Codepoints, CodepointsError> {
```

Comment:
this should be implemented on the UTF-8 borrowed type. It does not need to mutate the backing buffer.

Reply:
I implemented this on `Utf8Str` instead in b8729b7.